content/en/docs/setup/getting-started/index.md

```diff
@@ -342,16 +342,16 @@ by viewing the Bookinfo product page using a browser.
 Istio integrates with [several](/docs/ops/integrations) different telemetry applications. These can help you gain
 an understanding of the structure of your service mesh, display the topology of the mesh, and analyze the health of your mesh.
 
-Use the following instructions to deploy the [Kiali](/docs/ops/integrations/kiali/) dashboard.
+Use the following instructions to deploy the [Kiali](/docs/ops/integrations/kiali/) dashboard, along with [Prometheus](/docs/ops/integrations/prometheus/), [Grafana](/docs/ops/integrations/grafana), and [Jaeger](/docs/ops/integrations/jaeger/).
```

Comment:
Can you also update the expected output on lines 78-83 to not say that the addons are installed.

Reply:
done